common/djangoapps/monkey_patch/django_db_models_options.py

```diff
@@ -0,0 +1,45 @@
+"""
+Monkey patch implementation of the following _expire_cache performance improvement:
+
+https://github.com/django/django/commit/7628f87e2b1ab4b8a881f06c8973be4c368aaa3d
+
+Remove once we upgrade to a version of django which includes this fix natively!
```

Comment:
Nit: It'd be nice to state "This fix is not in an official Django release as of 1.8.x and 1.9.x"